crates/proto/Cargo.toml

```diff
@@ -31,14 +31,17 @@ maintenance = { status = "actively-developed" }
 
 [features]
 dns-over-tls = []
-dns-over-rustls = ["dns-over-tls", "rustls", "rustls-pemfile", "tokio-rustls", "webpki", "tokio-runtime"]
+dns-over-rustls = ["dns-over-tls", "rustls", "rustls-pemfile", "tokio-rustls", "tokio-runtime"]
 dns-over-native-tls = ["dns-over-tls", "native-tls", "tokio-native-tls", "tokio-runtime"]
 dns-over-openssl = ["dns-over-tls", "openssl", "tokio-openssl", "tokio-runtime"]
 
 dns-over-https-rustls = ["dns-over-https"]
-dns-over-https = ["bytes", "h2", "http", "dns-over-rustls", "webpki-roots", "tokio-runtime"]
+dns-over-https = ["bytes", "h2", "http", "dns-over-rustls", "tokio-runtime"]
 
-dns-over-quic = ["quinn", "rustls/quic", "dns-over-rustls", "bytes", "webpki-roots", "tokio-runtime"]
+dns-over-quic = ["quinn", "rustls/quic", "dns-over-rustls", "bytes", "tokio-runtime"]
+
+webpki-roots = ["dep:webpki-roots"]
```

Comment:
As written, this doesn't make much sense to me. Do you intend to expand the list of features that `webpki-roots`/`native-certs` requires in future PRs?

Reply:
> As written, this doesn't make much sense to me.

Sorry, not sure where you are coming from here.

I didn't make it require anything else on purpose because it's a shared feature between `dns-over-tls`, `dns-over-quic` and so on.

How about renaming it to `dns-over-rustls-webpki-root`?
Or did you specifically mean we should just require `dns-over-rustls` here?

Reply:
Sorry, I meant it's unnecessary to redirect the optional webpki-roots dependency through a `webpki-roots` feature.

Reply:
Ah sorry, yeah, removed it.